lib/prepare_business_uk_leaving_eu.yaml

```diff
@@ -85,45 +78,47 @@ pages:
           - "rail"
           - "warehousing-and-support-for-transportation"
           - "water-transport-maritime-ports"
-  employ_eu_citizens:
+  doing_business_in_the_eu:
     show_in_qa: true
     question_type: "single_wrapped"
-    question: "Do you employ anyone from another EU or EFTA country?"
+    question: "Do you import or do business abroad?"
     description: |
-      <p>Check which <a href="/eu-eea">countries are in the EU</a>. EFTA countries also include Norway, Switzerland, Iceland and Liechtenstein.</p>
-      <p>Check <a href="/employment-status">who counts as an employee</a>.</p>
-  doing_business_in_the_eu:
+    <p>Doing business abroad includes exporting and going for meetings or taking goods to a trade fair.</p>
+  employ_eu_citizens:
     show_in_qa: true
     question_type: "single_wrapped"
-    question: "Do you import or do business abroad?"
+    question: "Do you employ anyone from another European country?"
     description: |
-      <p>Doing business abroad includes going for meetings or taking goods to a trade fair.</p>
+      <p>This includes <a href="/eu-eea">countries that are in the EU</a> and Norway, Switzerland, Iceland and Liechtenstein.</p>
   regulations_and_standards:
     show_in_qa: true
     question_type: "single_wrapped"
     question: "Do you sell products and goods in the UK?"
+    description: |
+      <p>Products and goods only include physical items that you sell or trade.</p>
   personal_data:
     show_in_qa: true
     question_type: "single_wrapped"
-    question: "Do you use personal information that you store or get from the EU?"
+    question: "Do you use personal information that you store or get from another EU country?"
     description: |
       <p>Personal information includes customers’ addresses, staff working hours or information you give to a delivery company.</p>
+      <p>It also includes any data you send to another organisation in the EU, such as when you visit their website or online service.</p>
       <p>Skip this question if you’re not sure how you deal with personal information.</p>
   intellectual_property:
     show_in_qa: true
     question_type: "single_wrapped"
     question: "Does your business use or rely on intellectual property (IP) protection?"
     description: |
       <p>Intellectual property protection includes copyright, trade marks and patents.</p>
+    public_sector_procurement:
```

Comment:
This block needs to be level with `intellectual_property`.

Reply:
thanks 👍